mods/species/adherent/datum/species.dm

```diff
@@ -0,0 +1,218 @@
+/decl/species/adherent
+	name = SPECIES_ADHERENT
+	name_plural = "Adherents"
+
+	description = "The Vigil is a relatively loose association of machine-servitors, Adherents, \
```

Comment:
I feel like this needs to be trimmed like other species, but I'm not sure how you could reduce it without making it unclear what the adherent are.